src/base_controller.coffee

```diff
@@ -14,8 +14,9 @@ class BaseController
   getActions: ({only, except} = {}) ->
     only = [].concat only ? []
     except = [].concat except ? []
-    for action of this
-      continue if BaseController::[action]?
+    for action in Object.getOwnPropertyNames @constructor.prototype
```

Comment:
For some reason `Object.keys` doesn't return the keys of ES6 classes. I'm not sure what magic they are doing but `Object.getOwnPropertyNames` seems to work